pyhap/accessory_driver.py

```diff
@@ -642,16 +642,18 @@ def persist(self):
         tmp_filename = None
         try:
             temp_dir = os.path.dirname(self.persist_file)
+            logger.debug(f"Creating temp persist file in {temp_dir!r}")
```

Comment:
Please use lazy formatting for loggers
https://pylint.readthedocs.io/en/stable/user_guide/messages/warning/logging-fstring-interpolation.html

Reply:
fixed it